src/index.js

```diff
@@ -1,4 +1,49 @@
 /** @format */
 
-import _ from "lodash";
-import "./style.css";
+// import _ from 'lodash';
+import './style.css';
+
+const todoListContainer = document.querySelector('#todo-list-container');
+class Todo {
+  constructor() {
+    this.userTodo = [
+      {
+        description: 'Double-tab to edit',
+        completed: false,
+      },
+      {
+        description: "Drag'n drop to reorder your list",
+        completed: false,
+      },
+      {
+        description: 'Manage all your lists in one place',
+        completed: false,
+      },
+      {
+        description: 'Resync to clear out the old',
+        completed: false,
+      },
+    ];
+  }
```

Comment:
- [ ] I will encourage you to add an index to it so it would look more like:

```
    this.userTodo = [
      {
        description: 'Double-tab to edit',
        completed: false,
        index: 1,
      },
      {
        description: "Drag'n drop to reorder your list",
        completed: false,
        index: 2,
      },
      {
        description: 'Manage all your lists in one place',
        completed: false,
        index: 3,
      },
      {
        description: 'Resync to clear out the old',
        completed: false,
        index: 4,
      },
    ];
```

This is important as it would be used to track each data in the array on the UI. You will work more with this in React.